testerum-backend/testerum-runner/testerum-runner-junit/src/test/java/TesterumTestFactory.java

```diff
@@ -0,0 +1,14 @@
+import com.testerum.runner.junit5.TesterumJunitTestFactory;
+import java.util.List;
+import org.junit.jupiter.api.DynamicNode;
+import org.junit.jupiter.api.TestFactory;
+
+public class TesterumTestFactory {
```

Comment:
maybe rename this to "TesterumJUnitTest"? then we don't need to add "*TestFactory" as a test file name pattern in pom.xml